### PR TITLE
Add events for health checks

### PR DESCRIPTION
**Pull request by pfeuffer**
## Proposed changes

This adds events for health check results. 

### Your checklist for this pull request

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] PR is well described and the description can be used as commit message on squash
- [x] Target branch is not master (in most cases develop should bet the target of choice) 
- [x] Code does not conflict with target branch
- [x] New code is covered with unit tests
- [x] [Changelog entry file](https://github.com/scm-manager/changelog#changelog-entry-files) created in `gradle/changelog`
- [x] Definition of Done's fulfilled: [DoD](https://github.com/scm-manager/scm-manager/blob/develop/docs/en/development/definition-of-done.md) // [UI DoD](https://github.com/scm-manager/scm-manager/blob/develop/docs/en/development/ui-dod.md)
- [ ] Documentation updated (only necessary for new features or changed behaviour)

### Checklist for branch merge request (not required for forks)

- [ ] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [ ] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

### CODE REVIEW

File: src/main/js/events/HealthCheckFailureEvent.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      {t("scm-landingpage-plugin.myevents.healthCheckFailure.title", { repository: event.repository })}
+    </strong>
+  );
+
+  return (
+    <CardColumnSmall
+      link={link}
+      avatar={<Icon name={"exclamation-triangle"} className="fa-2x" />}
+      contentLeft={content}
+      contentRight={""}
```
Comment (eheimbuch) [inline src/main/js/events/HealthCheckFailureEvent.tsx:49]: Is this required?

File: src/main/js/events/HealthCheckSucceededEvent.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      {t("scm-landingpage-plugin.myevents.healthCheckSucceeded.title", { repository: event.repository })}
+    </strong>
+  );
+
+  return (
+    <CardColumnSmall
+      link={link}
+      avatar={<Icon name={"thumbs-up"} className="fa-2x" />}
+      contentLeft={content}
+      contentRight={""}
```
Comment (eheimbuch) [inline src/main/js/events/HealthCheckSucceededEvent.tsx:49]: Required?

File: src/main/resources/locales/de/plugins.json [hunk 3]
```diff
@@ -65,6 +65,14 @@
...
       "pluginCenterNotAvailable": {
         "title": "Plugin-Center nicht erreichbar"
+      },
+      "healthCheckFailure": {
+        "title": "Fehlerhaft: {{repository}}",
+        "description": "Die Überprüfung des Repositories ist fehlgeschlagen"
+      },
+      "healthCheckSucceeded": {
+        "title": "In Ordnung: {{repository}}",
+        "description": "Die Überprüfung des Repositories hat keine Fehler mehr festgestellt"
```
Comment (eheimbuch) [inline src/main/resources/locales/de/plugins.json:75]:
```suggestion
      "healthCheckFailure": {
        "title": "Fehlerhaft: {{repository}}",
        "description": "Die Integritätsprüfung des Repositories ist fehlgeschlagen"
      },
      "healthCheckSucceeded": {
        "title": "Fehlerfrei: {{repository}}",
        "description": "Die Integritätsprüfung des Repositories hat keine Fehler festgestellt"
```

**Review by eheimbuch — CHANGES_REQUESTED** [review]

File: src/main/js/events/HealthCheckSucceededEvent.tsx (hunk 2 above)
Comment (pfeuffer) [reply src/main/js/events/HealthCheckSucceededEvent.tsx:49]: It's a required attribute

File: src/main/js/events/HealthCheckFailureEvent.tsx (hunk 1 above)
Comment (pfeuffer) [reply src/main/js/events/HealthCheckFailureEvent.tsx:49]: It's a required attribute

**Pull request merged**